relay/network_graph/graph.py

```diff
@@ -353,3 +377,22 @@ def mediated_transfer(self, source, target, value):
             target = source
         assert fees == cost
         return cost
+
+    def triangulation_transfer(self, source, target_reduce, target_increase, value):
+        """simulate triangulation transfer off chain"""
+        cost, path = self.find_path_triangulation(source, target_reduce, target_increase, value)
+        assert path[0] == source
+        assert path[-1] == source
+        path = list(reversed(path))
+        fees = 0
+        if value < 0:
```

Comment:
What is this? Isnt this just silently droping an error with a weird outcome?

Reply:
No, it was for when we had the functionality of triangulation with a negative value (which cannot work if we have no means of making transfer without fees, as already discussed)